src/router/index.js

```diff
@@ -24,14 +25,15 @@ export function createRouter() {
       {path:'/audiobait',component:AudioBaitView, meta: {requiresAuth: true}},
       {path:'/audio',component:AudioView, meta: {requiresAuth: true}},
       {path:'/devices',component:DevicesView, meta: {requiresAuth: true}},
+      {path:'/devices/:devicename', name: 'device', component: DeviceView, meta: {requiresAuth: true}},
       {path:'/error',component:ErrorView, meta: {requiresAuth: true}},
       {path:'/groups', name: 'groups', component:GroupsView, meta: {requiresAuth: true}},
       {path:'/groups/:groupname', name: 'group', component: GroupView, meta: {requiresAuth: true}},
       {path:'/', name: 'home', component:HomeView, meta: {requiresAuth: true}},
       {path:'/login', name: 'login', component:LoginView},
       {path:'/recordings',component:RecordingsView, meta: {requiresAuth: true}},
       {path:'/register',component:RegisterView},
-      {path:'/video/:id',component:VideoView}
+      {path:'/video/:id',component:VideoView, meta: {requiresAuth: true}}
```

Comment:
if `requiresAuth` something we control? If so, should it perhaps default to true? (for both safety and convenience)

Reply:
Agreed, see: https://github.com/TheCacophonyProject/cacophony-web-vuex/issues/15